ledger/accountdb.go

```diff
@@ -552,10 +591,6 @@ func accountsInit(tx *sql.Tx, initAccounts map[basics.Address]basics.AccountData
 		for addr, data := range initAccounts {
 			_, err = tx.Exec("INSERT INTO accountbase (address, data) VALUES (?, ?)",
 				addr[:], protocol.Encode(&data))
-			if err != nil {
-				return err
-			}
```

Comment:
I think that we do want to have this. Otherwise, the `totals.AddAccount` might get overflowed, and the function would return `fmt.Errorf("overflow computing totals")`, while hiding the true error.